jenkins-slaves/jenkins-slave-ruby/Dockerfile

```diff
@@ -14,7 +14,7 @@ It is simple, straight-forward, and extensible." \
   PATH=$PATH:/home/jenkins/bin \
   PROMPT_COMMAND=". /opt/app-root/etc/scl_enable" \
   HOME=/home/jenkins \
-  ORIGIN_CLIENT=https://mirror.openshift.com/pub/openshift-v3/clients/3.11.50/linux/oc.tar.gz
+  ORIGIN_CLIENT=https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.4/linux/oc.tar.gz
```

Comment:
@tom-stockwell ; can you change the version to be an env? i.e.: OC_VERSION

this is how 99% of the other images do it, just to keep things consistent.

Reply:
Sure

Reply:
I noticed that `utilities/ubi8-bats/Dockerfile` uses an `ARG` while `tool-box/Dockerfile` uses an `ENV`. Any preference one way or the other?

Reply:
don't mind, the build process which releases these images doesn't pass in the value as an arg, so there isn't any benefit either way